ddp-workspace/projects/ddp-sdk/src/lib/components/login/auth0-code-callback.component.ts

```diff
@@ -32,14 +32,20 @@ export class Auth0CodeCallbackComponent implements OnInit, OnDestroy {
         if (authCode) {
             this.log.logEvent(this.LOG_SOURCE, 'Will login to ' + this.configuration.localRegistrationUrl + ' using code ' + authCode);
 
-            const tempSession = this.session.isTemporarySession() ? this.session.session : null;
+            const value = sessionStorage.getItem('localAuthParams') || "{}";
+            const params: object = JSON.parse(value);
+            sessionStorage.removeItem('localAuthParams');
+
             const registrationPayload = {
                 auth0ClientId: this.configuration.auth0ClientId,
                 studyGuid: this.configuration.studyGuid,
                 auth0Code: authCode,
                 redirectUri: this.configuration.auth0CodeRedirect,
-                ...(tempSession && {
-                    tempUserGuid: tempSession.userGuid
+                ...(params['temp_user_guid'] && {
```

Comment:
Nice catch--I had forgotten about local registration!  When we move to app engine, it should be straightforward to spin up stuff from a branch on-the-fly (albeit still sharing GCP databases), which might overall be faster/simpler than our current special case local registration setup.